apis/projects/v1alpha1/project_types.go

```diff
@@ -148,6 +148,14 @@ type ContainerExpirationPolicyAttributes struct {
 	NameRegex *string `url:"name_regex,omitempty" json:"name_regex,omitempty"`
 }
 
+// Struct representing the secret name and the secret namespace to use when importing a project with secret
+type ImportUrlSecretRef struct {
+	Name        string `json:"secretName"`
+	Namespace   string `json:"secretNamespace"`
+	UsernameKey string `json:"username" default:"username"`
+	PasswordKey string `json:"password" default:"password"`
```

Comment:
The `default` tag has no effect. To configure default values for Kubebuilder the comment markers need to be used: https://book.kubebuilder.io/reference/markers/crd-validation